packages/swap/src/quoting/Quoter.ts

```diff
@@ -207,11 +214,11 @@ export default class Quoter {
         this.inflightRequests.delete(request.request_id);
       };
 
-      sub = this.quotes$.subscribe(({ marketMaker, quote }) => {
+      sub = this.quotes$.subscribe(({ marketMaker, quote, beta }) => {
```

Comment:
are we allowing them to pass the `beta` state?

Reply:
ah nvm, just saw that we are setting the beta in the socket session based on the db value. All good